src/lib/components/Cytoscape.react.js

```diff
@@ -899,7 +908,51 @@ Cytoscape.propTypes = {
     /**
      * Toggles intelligent responsive resize of Cytoscape graph with viewport size change
      */
-    responsive: PropTypes.bool
+    responsive: PropTypes.bool,
+
+    /**
+     * Displays a context menu on right click. Requires extra layouts loaded.
+     * Accepts a list of dictionaries, each of which describes a context
+     * menu option. Options are rendered in the order presented.
+     */
+    cxtmenu: PropTypes.arrayOf(
+        PropTypes.exact({
+            /** ID associated with option. */
+            id: PropTypes.string,
+            /** 
+             * Determines which Cytoscape elements the option is attached to. Takes in a Cytoscape selector
+             * (see Cytoscape documentation for more information). Examples of valid selectors include node,
+             * edge, and core.
+             */
+            selector: PropTypes.string,
+            /** Label assigned to option. */
+            content: PropTypes.string,
+            /** Hover tooltip text assigned to option. */
+            tooltipText: PropTypes.string,
+            /** Toggles option disabled (greyed out). */
+            disabled: PropTypes.bool
+        })
+    ),
+
+    /**
+     * Dictionary returned when you a context menu option is selected. Read-only.
```

Comment:
```suggestion
     * Dictionary returned when a context menu option is selected. Read-only.
```
The source of the typo @mtwichan pointed out